### DISCUSSION TITLE
Hello, some suggestions, hope to get discussion and help.

### DISCUSSION THREAD

**Discussion by bit365** [microsoft/fluentui-blazor#1917] [Q&A] [answered]
1. The fluent 2.0 design has been released. Does fluentui-blazor currently support version 2.0?

2. Is it possible to provide a colorselect component to support color palette selection?

**Comment by vnbaaij** [comment, marked as answer]
Hi, these answers can easily be found by doing some searches here and/or on our demo/docs site...

1) No, the components currently wrap the FLuent UI Web Components v2 and those support Fluent Design 1
2) Yes, but the accent color will always use a recipe to determine accessible colors so you will probably not get an exact match.
